astropy/visualization/wcsaxes/__init__.py

```diff
@@ -0,0 +1,11 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+# The following few lines skip this module when running tests (and have no
```

Comment:
This only skips it if matplotlib is not available, correct? If so, update the comment to say so.